config/initializers/omniauth.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
-  provider :github,
-    Rails.application.credentials.github[:client_id],
-    Rails.application.credentials.github[:client_secret]
+  if Rails.application.credentials.github
+    provider :github,
+      Rails.application.credentials.github[:client_id],
+      Rails.application.credentials.github[:client_secret]
+  end
```

Comment:
- https://fly.io/docs/rails/getting-started/existing/#access-to-environment-variables-at-build-time